From eb89cd344535706c6194359bd1f1fed50162a041 Mon Sep 17 00:00:00 2001
From: Andrey Zhizhikin <andrey.z@gmail.com>
Date: Tue, 1 Sep 2020 08:24:52 +0000
Subject: [PATCH] fsl_parser.h: fix compiler error with -fcommon

When -fcommon is set in GCC10 configuration, following error is produced
during linking:
./git/ext-includes/fsl_parser.h:598: multiple definition of `TrackExtTagIndex';
./git/ext-includes/fsl_parser.h:598: first defined here

Remove unused enum name to resolve multiple definitions, it is not used
anywhere in the code base.

Signed-off-by: Andrey Zhizhikin <andrey.z@gmail.com>
---
 ext-includes/fsl_parser.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ext-includes/fsl_parser.h b/ext-includes/fsl_parser.h
index d3aae20..c037c9a 100755
--- a/ext-includes/fsl_parser.h
+++ b/ext-includes/fsl_parser.h
@@ -595,7 +595,7 @@ enum{
     FSL_PARSER_TRACKEXTTAG_CRYPTO_ENCRYPTED_BYTE_BLOCK,
     FSL_PARSER_TRACKEXTTAG_CRYPTO_SKIP_BYTE_BLOCK,
     FSL_PARSER_TRACKEXTTAG_CRYPTO_IV,
-}TrackExtTagIndex;
+};
 
 typedef struct _SeiPosition
 {
-- 
2.17.1

